platform/darwin/test/MGLStyleValueTests.swift

```diff
@@ -1,8 +1,53 @@
 import XCTest
 import Mapbox
 
-
+#if os(iOS) || os(watchOS) || os(tvOS)
+typealias ColorType = UIColor
+#elseif os(macOS)
+typealias ColorType = NSColor
+#endif
+    
 extension MGLStyleValueTests {
+    func assertColorsEqualWithAccuracy(_ actual: ColorType, _ expected: ColorType, accuracy: Float = 0.1) {
```

Comment:
Just an observation, not a request for change: Swift lets you overload methods like in C++. So you could overload `XCTAssertEqualWithAccuracy(_:_:accuracy:)` with two `MGLColor` arguments. Copying the existing `XCTAssertEqualWithAccuracy(_:_:accuracy:)` signature, you’d wind up with:

```swift
func XCTAssertEqualWithAccuracy<T : FloatingPoint>(_ color1: MGLColor, _ color2: MGLColor, accuracy: T, _ message: @autoclosure () -> String = "", file: StaticString = #file, line: UInt = #line)
```

With this fancy signature, you could pass `file` and `line` into the calls to the normal `XCTAssertEqualWithAccuracy(_:_:accuracy:)` and get a failure message that refers to the original line number.